### PR TITLE
Docs: information about extensions

### LINKED ISSUE #131: Extensions from Portainer not Appearing
I installed extensions into /portainer/Files/AppData/Config/FreshRSS/www/freshrss/extensions

Inside of the folder is the "README.md" file, stating the extension should be placed into this file.

However, the extensions are not appearing under the extensions tab on my FreshRSS page.

Why can I do about this?

### PR DESCRIPTION
**Pull request by math-GH**
Closes https://github.com/FreshRSS/Extensions/issues/131

Changes proposed in this pull request:

- added information about extensions to the docs

### CODE REVIEW

File: docs/en/admins/15_extensions.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Extensions
+
+Not every feature is relevant for the most users, but it some special features are relevant for the user. That is why FreshRSS is extendable.
```
Comment (Frenzie) [inline docs/en/admins/15_extensions.md]:
```suggestion
Not every feature is relevant for all users, but some special features are relevant to the right person. That is why FreshRSS is extendable.
```

File: docs/en/admins/15_extensions.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+There are a bunch of "official" extensions (supported and published by FreshRSS development team and community) and "Community" extensions (developed and published individually by thiry-party).
```
Comment (Frenzie) [inline docs/en/admins/15_extensions.md]: A bunch of is very informal.
```suggestion
There are some "official" extensions (supported and published by the FreshRSS development team and community) and "community" extensions (developed and published individually by third-party developers).
```

File: docs/en/admins/15_extensions.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+Important: Do not delete or overwrite the existing files `./extensions/.gitignore` and `./extensions/README.md` 
+
+## How to enable/disable and manage
+
+See in the front end: configuration menu `Configuration/Extensions`
+
+### User extensions
+
+Every user has to manage the extensions by them self. The configuration via the cog icon is valid only for the user, not for other users.
```
Comment (Frenzie) [inline docs/en/admins/15_extensions.md]: A cog is just one little tooth. The whole thing is a cogwheel or a gear. I prefer gear.
```suggestion
Every user has to manage the extensions by themselves. Configuration via the gear icon is valid only for that user, not for other users. 
```

File: docs/en/admins/15_extensions.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+### User extensions
+
+Every user has to manage the extensions by them self. The configuration via the cog icon is valid only for the user, not for other users. 
+
+`metadata.json`: "type": "user"
+
+### System extensions
+
+Only administrators can enabled/disable system extensions. The configuration via the cog icon is valid for every user.
```
Comment (Frenzie) [inline docs/en/admins/15_extensions.md]:
```suggestion
Only administrators can enable/disable system extensions. The configuration via the gear icon is valid for every user. 
```

File: docs/en/admins/15_extensions.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+Only administrators can enabled/disable system extensions. The configuration via the cog icon is valid for every user. 
+
+`metadata.json`: "type": "system"
+
+### pre installed extensions (core extensions)
+
+See folder: `.lib/core-extensions`
+
+2 system extensions are already pre installed: `Google-Groups` and `Tumblr-GDPR`.
```
Comment (Frenzie) [inline docs/en/admins/15_extensions.md]:
```suggestion
Two system extensions are already pre-installed: `Google-Groups` and `Tumblr-GDPR`.
```

File: docs/en/admins/15_extensions.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+Important: Do not install you chosen extensions here!
```
Comment (Frenzie) [inline docs/en/admins/15_extensions.md]:
```suggestion
Important: do not install your chosen extensions here!
```

**Review by Frenzie — COMMENTED** [review]
Looks pretty good! :+1:

File: docs/en/admins/15_extensions.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+## How to enable/disable and manage
+
+See in the front end: configuration menu `Configuration/Extensions`
+
+### User extensions
+
+Every user has to manage the extensions by themselves. Configuration via the gear icon is valid only for that user, not for other users.
+
+`metadata.json`: "type": "user"
```
Comment (Frenzie) [inline docs/en/admins/15_extensions.md]: (same for the others)
````suggestion
```
`metadata.json`: "type": "user"
```
````

**Comment by math-GH** [comment]
@Frenzie Which language would you suggest?
grafik

**Comment by Frenzie** [comment]
js if json's not an option.

**Review by Frenzie — APPROVED** [review]
Definitely clearer.

**Pull request merged**